blocks/tabs/tabs.js

```diff
@@ -65,8 +65,11 @@ export default async function decorate(block) {
     tab.remove();
   });
   block.prepend(tablist);
-  block.querySelectorAll('.block.media').forEach((mediaBlock) => {
+  const mediaBlocks = Array.from(block.querySelectorAll('.block.media'));
+  // eslint-disable-next-line no-restricted-syntax
+  for (const mediaBlock of mediaBlocks) {
     decorateBlock(mediaBlock);
-    loadBlock(mediaBlock);
-  });
+    // eslint-disable-next-line no-await-in-loop
+    await loadBlock(mediaBlock);
```

Comment:
@ravverma if `loadBlock` has some API call, we might be stuck at those API response time. 
Is that something we already accounted while applying this await ?

Reply:
@anagarwa 
the block loading(media) using `observe` it doesn't stuck for the response.